components/Footer.js

```diff
@@ -18,6 +18,8 @@ import StyledLink from './StyledLink';
 import StyledTooltip from './StyledTooltip';
 import ExternalLinkNewTab from './ExternalLinkNewTab';
 
+import languages from '../src/constants/locales';
```

Comment:
We got rid of the `src` folder since the initial spec. Having this file in `src` is the reason why the build is crashing. Moving it to `constants/locales.js` should do the job

Reply:
@Betree Did you mean **lib/constants** or **components/constants**?

Reply:
@leonardohof `lib/constants`, the one that already exists today

Reply:
Ok, changed!